### PR TITLE
docs(tutorial): Add introduction and treeshaking tutorial

### PR DESCRIPTION
**Pull request by mhevery**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/qwik/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/qwik",
-  "version": "0.0.26",
+  "version": "0.0.25",
```
Comment (adamdbradley) [inline packages/qwik/package.json:3]: Can you change this back to 26?

File: packages/qwik/src/optimizer/core/Cargo.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ authors = ["Manu <manu@builder.io>"]
 name = "qwik-core"
 version = "0.2.0"
 edition = "2021"
-repository = "https://github.com/builder.io/qwik"
+repository = "https://github.com/builderio/qwik"
```
Comment (adamdbradley) [inline packages/qwik/src/optimizer/core/Cargo.toml:6]: Out of curiosity, why was this changed?

Comment (mhevery) [reply packages/qwik/src/optimizer/core/Cargo.toml:6]: Ohh it was a global replace. :-) But the old URL is 404 and new URL is the correct one.

File: packages/qwik/package.json (hunk 1 above)
Comment (mhevery) [reply packages/qwik/package.json:3]: reverted
